### PR TITLE
Write up how to deploy analytical clusters

### PR DESCRIPTION
**Pull request by nick-giles-neo**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-807.surge.sh

File: modules/ROOT/pages/clustering/setup/analytical.adoc [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+[[cluster-example-create-databases-on-cluster]]
+=== Create new databases in the cluster
+
+As mentioned in the xref:clustering/introduction.adoc[Introduction], a server in a cluster can either host a database in primary or secondary mode.
+The primaries provide write operations and fault tolerance if there are more than one of them.
+The secondaries provide somewhere for read queries to run, including the analytical queries for this example.
+
+.Create a new database with one primary and two secondaries
+====
+In the `system` database on the writer from the previous example, execute the following Cypher command to create a new database:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/clustering/setup/analytical.adoc:128]: There is no _writer_ specified in the previous example. Do you mean to say to execute on any of the primaries? They are essentially all writers, right?

Comment (nick-giles-neo) [reply modules/ROOT/pages/clustering/setup/analytical.adoc:128]: Yep, that's pretty much a copy-paste failure. I'll just say 'on the `system` database'.

File: modules/ROOT/pages/clustering/setup/analytics-cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,272 @@
+:description: This section describes how to deploy a special case Neo4j cluster for analytic queries.
+[role=enterprise-edition]
+[[clustering-analytics]]
+= Deploy an analytics cluster
+
+In analytic use cases, the analytic queries can be intensive enough to warrant running them on separate servers, away from the servers handling the transactional workload.
+This example shows how to set up a cluster to achieve that.
+Information on using Neo4j's graph data science library in a cluster can be found here: link:https://neo4j.com/docs/graph-data-science/current/production-deployment/neo4j-cluster/[Neo4j Graph Data Science Library Manual -> GDS with Neo4j cluster].
+
+The analytics cluster can be set up in two ways, with fault tolerance or without fault tolerance.
```
Comment (vnickolov) [inline modules/ROOT/pages/clustering/setup/analytics-cluster.adoc:10]: Maybe add a note here that the GDS library does not support fault tolerance, be crystal clear and don't leave anything open to interpretation 🤔

**Review by vnickolov — APPROVED** [review]
Generally looks pretty good, great job 🙇 
I have left a couple of comments to add a bit more clarification around GDS but will leave it to you to decide if you wan to take them into an account.

Thank you for this 🙇

File: modules/ROOT/pages/clustering/setup/analytics-cluster.adoc (hunk 2 above)
Comment (vnickolov) [reply modules/ROOT/pages/clustering/setup/analytics-cluster.adoc:10]: Alternatively we can update the documentation at our end to point this out 🤔

**Pull request merged**